### ISSUE TITLE
Parsing/Rendering Issue

### ISSUE THREAD

**Issue by aleemb** [OscarGodson/EpicEditor#69]
The editor doesn't work for it's own [readme.md](https://raw.github.com/OscarGodson/EpicEditor/master/README.md). After pasting that file into the editor, it doesn't allow scrolling below the **constructor** section. In some places the double-asterisk are not being converted to bold for the same file.

**Comment by OscarGodson** [comment]
Yep. This was because in the alpha/prototype we used a port GitHub Flavored MD port of Showdown.js by @isaacs which we tested out and it did fail miserably unfortunately when trying to handle Markdown with HTML. We've switched to Marked by @chjj (https://github.com/chjj/marked) and it appears to be working a lot better and our README no long breaks it, but we need to do more extensive testing to be sure.

This is a duplicate of #55 which @dadambickford first reported and why we switched to Marked which @johnmdonahue found so I'm closing it. Feel free to use and test on that. It's just in the 0.1.0 branch (please note it's our WIP branch)

In short, we know, and it's fixed in the working branch and will be in the next release we're working hard on :)
